src/select.tsx

```diff
@@ -365,7 +365,9 @@ const Select: React.FC<SelectProps> = ({
                             <option value="" style={{display: 'none'}} />
                         )}
                         {options.map(({value: val, text}) => (
-                            <option key={val} value={val}>
+                            // Set color: 'initial' to avoid wrong text color in some browsers when using dark mode.
```

Comment:
Styling `<option/>` element is really tricky, most browsers completely ignore custom styles. The only browser that had the issue reported in the ticket is (afaik) Chrome. I've verified that this change fixes the problem in that browser and that none of the other ones are affected:

- FF / Chrome in Linux :heavy_check_mark:  
- Safari in a VM :heavy_check_mark: 
- Safari / Chrome in iOS :heavy_check_mark: 
- Chrome in Android :heavy_check_mark: 
- Chrome in Windows: pending